application/language/german/general_words_lang.php

```diff
@@ -7,9 +7,20 @@
 $lang['notice_turn_the_radio_on'] = 'Du hast heute noch keine QSOs gemacht; Zeit, das Funkgerät einzuschalten!';
 
 $lang['general_word_important'] = 'Wichtig';
+$lang['general_word_info'] = 'Info';
+$lang['general_word_choose_file'] = 'Datei wählen';
 
 $lang['general_word_date'] = 'Datum';
 $lang['general_word_time'] = 'Zeit';
+$lang['general_word_none'] = 'None';
+$lang['general_word_name'] = 'Name';
+$lang['general_word_location'] = 'Die Stelle';
```

Comment:
I prefer using "Stelle" alone instead of prefixing it with an article.

Reply:
OK.